src/client/game/game.js

```diff
@@ -3,33 +3,79 @@ import { connect } from 'react-redux';
 
 import AnswerInput from './answer-input';
 
-import { setLeader } from './game-actions';
+import { setLeader, getConsonantDispatch, getVowelDispatch, 
+        disableConsonantDispatch, disableVowelDispatch } from './game-actions';
 
-import { setLeaderDispatch } from '../services/cloak-service';
+import { setLeaderDispatch, setConsonantDispatch, messageGetConsonant, 
+        setVowelDispatch, messageGetVowel, setDisableConsonantDispatch,
+        setDisableVowelDispatch } from '../services/cloak-service';
+
+import storageService from '../services/storage-service';
 
 export class Game extends Component {
     componentWillMount() {
         setLeaderDispatch(this.props.setLeader);
+        setConsonantDispatch(this.props.getConsonantDispatch);
```

Comment:
All of this set{_Something_}Dispatch is making me nervous - I don't get it. Would appreciate an in person explanation at some point